index.html

```diff
@@ -329,8 +345,30 @@
       cmds.join('<br/>&nbsp;&nbsp;/');
     box.scrollTop = box.scrollHeight - 10;
   },
+  rosterize: function() {
+    var all = box.innerHTML.split("<br>").
+              map(function(s) { return !!s ? s.split(":")[0] : ""; });
+    all.forEach(addSpeaker);
+  },
```

Comment:
simply takes the current chat log, and divines a speaker roster from it.  Helpful to me, but it can be removed.